.gitignore

```diff
@@ -11,6 +11,7 @@ private
 *.tmp
 .history
 .DS_Store
+.zed
```

Comment:
here because I use zed. Zed is also becoming popular so I thought it could make sense to acknowledge it and ignore in git